tasks/eva_2121/aggregate_table.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python
+import csv
+import glob
+import logging
+import os
+from argparse import ArgumentParser
+from collections import defaultdict
+from operator import itemgetter
+
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+
+assigned_number_variant = 0
+number_of_tempmongo_instances = 10
+eva_accession_path = ''
+
+logger = log_cfg.get_logger(__name__)
+
+
+def parse_application_properties(app_prop):
+    properties_dict = { }
+    with open(app_prop) as open_file:
+        for line in open_file:
+            if not line.strip() or line.strip().startswith('#'):
+                continue
+            sp_line = line.strip().split('=')
+            properties_dict[sp_line[0]] = sp_line[1]
+    return properties_dict
+
+
+def find_and_parse_properties_and_parse(properties_path):
+    property_files = glob.glob(os.path.join(properties_path, '*', 'release_phase1', '*', 'application.properties'))
```

Comment:
I would also add '*application.properties' because IIRC some properties files will have an assembly accession prepended to the name :(

Reply:
Done